pylint/pyreverse/vcgutils.py

```diff
@@ -152,73 +154,129 @@
     "anchor": 1,
     "horizontal_order": 1,
 }
-
+SHAPES: Dict[NodeType, str] = {
+    NodeType.PACKAGE: "box",
+    NodeType.CLASS: "box",
+    NodeType.INTERFACE: "ellipse",
+}
+ARROWS: Dict[EdgeType, Dict] = {
+    EdgeType.USES: dict(arrowstyle="solid", backarrowstyle="none", backarrowsize=0),
+    EdgeType.INHERITS: dict(
+        arrowstyle="solid", backarrowstyle="none", backarrowsize=10
+    ),
+    EdgeType.IMPLEMENTS: dict(
+        arrowstyle="solid",
+        backarrowstyle="none",
+        linestyle="dotted",
+        backarrowsize=10,
+    ),
+    EdgeType.ASSOCIATION: dict(
+        arrowstyle="solid", backarrowstyle="none", textcolor="green"
+    ),
+}
+ORIENTATION: Dict[Layout, str] = {
```

Comment:
Maybe a function that return the proper value in the Layout enum (Orientation or rankdir) would reduce the number of thing to keep in mind when using the code.

Reply:
Hm, I don't know - we would just copy the content of the dictionaries into the ``Layout`` enum. Plus, the enum would then need to know the details of each backend (how the parameters have to look for ``dot``, ``vcg``, and ``puml`` - which is even more tricky as it only supports top to bottom and left to right, but not the other two). So every time a new backend is added, we would not only need to add the new printer class but also modify the ``Layout``.
This would be against the Open-Closed-Principle in my opinion.

Reply:
You're right I did not realize that :) !